docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -996,49 +1068,151 @@ The UI design consists of three parts:
-import { getFullnodeUrl } from '@mysten/sui/client';
-import { Theme } from '@radix-ui/themes';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { RouterProvider } from 'react-router-dom';
+### Setting up import aliases
 
-import { router } from '@/routes/index.tsx';
+First, set up import aliases to make the code more readable and maintainable. This allows you to import files using `@/` instead of relative paths.
 
-const queryClient = new QueryClient();
+Replace the content of `tsconfig.json` with the following:
 
-const { networkConfig } = createNetworkConfig({
-	localnet: { url: getFullnodeUrl('localnet') },
-	devnet: { url: getFullnodeUrl('devnet') },
-	testnet: { url: getFullnodeUrl('testnet') },
-	mainnet: { url: getFullnodeUrl('mainnet') },
-});
+{@inject: examples/trading/frontend/tsconfig.json}
+
+Replace the content of `vite.config.ts` with the following:
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
-	<React.StrictMode>
-		<Theme appearance="light">
-			<QueryClientProvider client={queryClient}>
-				<SuiClientProvider networks={networkConfig} defaultNetwork="testnet">
-					<WalletProvider autoConnect>
-						<RouterProvider router={router} />
-					</WalletProvider>
-				</SuiClientProvider>
-			</QueryClientProvider>
-		</Theme>
-	</React.StrictMode>,
-);
+{@inject: examples/trading/frontend/vite.config.ts}
+
+### Adding Tailwind CSS
+
+This app uses Tailwind CSS for styling. Run the following command to add Tailwind CSS and its dependencies:
+
+```bash
+pnpm add tailwindcss@latest postcss@latest autoprefixer@latest
+```
+
+or
+
+```bash
+yarn add tailwindcss@latest postcss@latest autoprefixer@latest
+```
+
+Next, generate the Tailwind CSS configuration file by running:
+
+```bash
+npx tailwindcss init -p
 ```
 
+Replace the content of `tailwind.config.js` with the following:
+
+{@inject: examples/trading/frontend/tailwind.config.js}
+
+Add the `src/styles/` directory and add `base.css`: 
+
+{@inject: examples/trading/frontend/src/styles/base.css}
+
+### Connecting your deployed package
+
+First, deploy your package via the [scripts in the api directory](#deployment).
+
+Then, create a `src/constants.ts` file and fill it with the following:
+
+{@inject: examples/trading/frontend/src/constants.ts}
+
+### Add helper functions and UI components
+
+Create a `src/utils/` directory and add `helpers.ts`: 
+
+{@inject: examples/trading/frontend/src/utils/helpers.ts}
+
+Create a `src/components/` directory and add the following components: 
+
+- `ExplorerLink.tsx`:
+
+{@inject: examples/trading/frontend/src/components/ExplorerLink.tsx}
+
+- `InfiniteScrollArea.tsx`:
+
+{@inject: examples/trading/frontend/src/components/InfiniteScrollArea.tsx}
+
+- `Loading.tsx`:
+
+{@inject: examples/trading/frontend/src/components/Loading.tsx}
+
+- `SuiObjectDisplay.tsx`:
+
+{@inject: examples/trading/frontend/src/components/SuiObjectDisplay.tsx}
+
+### Set up routes
```

Comment:
```suggestion
### Set up routing
```
I'm not sure if this is technically more accurate or if it even matters. Made the suggestion anyway :)